docs/source-fabric/advanced/model_parallel/fsdp.rst

```diff
@@ -142,7 +142,7 @@ We can specify a list of layer classes in the **wrapping policy** to inform FSDP
         # 3. Pass it to the FSDPStrategy object
         strategy = FSDPStrategy(auto_wrap_policy=policy)
 
-    PyTorch provides several of these functional policies under :mod:`torch.distributed.fsdp.wrap`.
```

Comment:
This was correct

See [pytorch/pytorch@main/torch/distributed/fsdp/wrap.py](https://github.com/pytorch/pytorch/blob/main/torch/distributed/fsdp/wrap.py)

Reply:
The same, PyTroch does not generate this as a module...

Reply:
Ok, then maybe we remove `:mod:` and just reference "torch.distributed.fsdp"

Reply:
the updated `:mod:'torch.distributed.fsdp'` is fine

Reply:
But that doesn't point to the wrapping utilities specifically

Reply:
Yes, unfortunately wrapping is in PT docs or you found it somewhere? Otherwise we need to add plain link to PT codebase....